nbs/index.ipynb

```diff
@@ -1,23 +1,52 @@
 {
```

Comment:
<p>this looks weird. There isnt neccessarily an edge between x and z so why does it appear in P?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>The string in the description didn't match the string in the code. fixed it.</p><p> </p>